connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -323,10 +331,6 @@ def __get_envs(self):
             os.path.exists, "/app/certs/cosign", f"/app/certs/cosign/{self.name}.crt"
         ):
             env["SSL_CERT_FILE"] = f"/app/certs/cosign/{self.name}.crt"
-        # Rekor support requires setting of COSIGN_EXPERIMENTAL
```

Comment:
This sounds like a new promise of Rekor support not being experimental, which I assume was documented somewhere. In that case, that disclaimer should be removed